### ISSUE TITLE
Latest 6.0.9 update won’t autoplay TV SHOWS

### ISSUE THREAD

**Issue by jaimsitecom** [123Venom/plugin.video.venom#106]
After updating to 6.0.9, tv episodes on kodi library wont ‘autoplay’ anymore when clicked. Had to go through tmdb helper context menu options to choose venom’s autoplay or select source to play TV show. 

But with movies in kodi’s library, autoplay works fine. 

hope this gets looked at and fixed? I already downgraded to 6.0.8, and it’s working great again. 

Also, a month ago, i tried to request a feature where it will be possible to lower maximum filesize of scraped filea down to 2gb instead of 5gb. :) i hope this is still possible on the next update. I know how to manually manipulate as per your instructions but kinda hassle to do it every update. Hehe 

anyways ty for this great kodi addon!

**Comment by 123Venom** [comment]
Your first issue is confirmed.  The new **country_code** check I added is not possible from library .strm files.  I will have have to add a try except block and you will not have that benefit, sorry it's just a limitation of using library integration and as always I neglected to test library playback.  This feature was added for a show like **Animal Kingdom** where many scene releases have the `US` country code in the magnet title so it failed our aggressive title checking and would not be included.  There's a few scattered shows we noted this new feature would also pick up a few additional links due to improperly adding country codes by the uploader. **Yellowstone**, **The Witcher** was such cases we noted.

Your 2nd item.  I actually did forget about this.  I've fixed item 1 and I'll also fix item 2 so both will be included in next release.
